typescript/packages/wsc-common/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
You need to re-run a top-level npm install. It does a slight modification in the top level package-lock.json.

Reply:
OK, it seems so from my tests ;)